modules/oe_content_news/oe_content_news.info.yml

```diff
@@ -5,7 +5,9 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - field_group:field_group
```

Comment:
We have to enable this in a update hook.

Reply:
Done